roles/installer/templates/config.yaml.j2

```diff
@@ -101,6 +101,11 @@ data:
     BROADCAST_WEBSOCKET_PORT = 8052
     BROADCAST_WEBSOCKET_PROTOCOL = 'http'
 
+    # Load LDAP BIND password from Kubernetes secret if define
+    {% if ldap_password_secret -%}
+    AUTH_LDAP_BIND_PASSWORD = "{{ ldap_bind_password }}"
```

Comment:
Hello and thanks for your contribution. Unfortunately it won't be _quite_ this easy. ConfigMaps should not be used to store sensitive information because they are not encrypted.

Luckily we already have a pattern for this. Check out these files:

https://github.com/ansible/awx-operator/blob/ebe5d1e2f4afd7fd329c18420947ff5840670293/roles/installer/templates/app_credentials.yaml.j2

https://github.com/ansible/awx-operator/blob/ebe5d1e2f4afd7fd329c18420947ff5840670293/roles/installer/templates/credentials.py.j2